Gemfile

```diff
@@ -2,6 +2,7 @@ source 'https://rubygems.org'
 ruby '2.2.2'
 
 gem "csvlint"
+gem "haml_lint"
```

Comment:
The tests will fail,so you need to change this line to `gem "haml_lint", "0.26.0"`.